### DISCUSSION TITLE
Blank screen (Black) when you lock your session in windows or the computer screen goes to sleep.

### DISCUSSION THREAD

**Discussion by Shaztech** [Lenochxd/WebDeck#21] [Ideas]
Is it possible to add a feature that turn the screen black (on the device webpage), when you lock your windows session or when monitor goes to sleep.

Thanks

**Comment by Lenochxd** [comment]
I added it to my todo list :)
it will be added in an update very soon

**Reply by Shaztech** [reply]
Thanks, I've made a Raspberry Pi enclosure (3D printed) with a 7 inch touch screen running in kiosk mode that run only Webdeck. So it's will be a nice option !

**Reply by Lenochxd** [reply]
Oh, it's you! I saw your email like 20 minutes ago, super cool, thanks for using webdeck :)

**Comment by Shaztech** [comment]
Also while your at it, I have 2 things that would make configuration a lot easier.
- Add the option to copy a button that already exist.
- When you edit a button. Be able to edit the command/path/sound etc. Not just the text and image.

**Reply by Lenochxd** [reply]
I already thought about the option to copy a button, it would be simple to implement but I had forgotten the idea so thank you!

As for editing a button, the implementation can be very complex, I have to rewrite how the buttons work.
In the meantime you can activate developer mode in the settings to be able to modify the command of a button :)

**Reply by Shaztech** [reply]
I just found another small bug, If you change the image of a button and the filename is the same as it was before. The image doesn't change (probably cached in a folder).

**Reply by Lenochxd** [reply]
Did you try reloading the page without cache?

**Comment by Shaztech** [comment]
I've got others thing to add if possible, I want to use the WebDeck interface for a table RPG game. 

- Add a repeat mode for the sound board button, so a background track can be played continously.
- Add a toggle mode on a sound board button, so you can start / stop a specific sound without stopping everything.

Thanks
